files/ceph-version-qemu.sh

```diff
@@ -59,7 +59,10 @@ for YUM_HISTORY_EVENT in $YUM_HISTORY_EVENTS; do
 done
 
 # Retrieve a list of all QEMU process IDs
-PIDS=$(pidof /usr/libexec/qemu-kvm)
+# The pidof command can be used when this bug is fixed 
+# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=926896
+# PIDS=$(pidof /usr/libexec/qemu-kvm)
+PIDS=$(ps aux | grep -v root |  grep qemu| awk -F' ' '{print $2}')
```

Comment:
pgrep -f /usr/libexec/qemu-kvm
 
 This is probably a simpler way to check.

Reply:
I agree. It looks better. I will push a new commit.

Reply:
Done.